### DISCUSSION TITLE
Specify latency via a URL parameter

### DISCUSSION THREAD

**Discussion by TofPlay** [mockoon/mockoon#1606] [Help] [answered]
Hi, 

I defined a root with Mockoon and I would like to pass the latency of the root on the url

Like: http://localhost:3000/product/1?latency=300ms

With Mockoon I would like to retrieve the latency parameter (like with {{lantency}}) and thus configure the latency of the root

**Comment by 255kb** [comment, marked as answer]
Hi, this is not yet possible. 
There is currently no plan to allow this kind of configuration through query params and I'm not sure it's something we want as the query params are used for other purposes. 
We have an Admin API that will be improved in the future to support this kind of configuration. I think it's a better candidate.
